docs/DeveloperGuide.md

```diff
@@ -728,26 +728,82 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 Given below are instructions to test the app manually.
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
-testers are expected to do more *exploratory* testing.
+testers are expected to do more *exploratory* testing. Alternative test cases with similar expected results are provided but are **not exhaustive**.
```

Comment:
```suggestion
testers are expected to do more *exploratory* testing. Alternative test cases with similar expected results are provided for each example test case but are **not exhaustive**.
```
Perhaps this would be clearer?